src/main/scala/io/getclump/package.scala

```diff
@@ -0,0 +1,15 @@
+package io
+
+package object getclump {
+
+  private[getclump] implicit val executionContext = scala.concurrent.ExecutionContext.global
```

Comment:
It would be possible to make it configurable by system properties (i.e. executorType, numberOfThreads, etc).

I'm not sure I understood the suggestion to use `implicitly`. It would require to change the methods signatures to receive the implicit any way, no?